platform/linux-generic/pktio/dpdk.c

```diff
@@ -1079,7 +1079,7 @@ static int dpdk_input_queues_config(pktio_entry_t *pktio_entry,
 				    const odp_pktin_queue_param_t *p)
 {
 	odp_pktin_mode_t mode = pktio_entry->s.param.in_mode;
-	odp_bool_t lockless;
+	uint8_t lockless;
```

Comment:
why is that needed?

Reply:
While packing the pkt_dpdk_t struct I changed pkt_dpdk_t.lockless_rx and pkt_dpdk_t.lockless_tx to uint8_t. I'm simply using the same type here.

Reply:
@matiaselo  yes, I see. While reading patches in that web interface sometime I miss some pieces of code.  Just above you changed that.